met-web/src/components/engagement/view/PreviewBanner.tsx

```diff
@@ -61,8 +43,17 @@ export const PreviewBanner = () => {
             }}
         >
             <Grid container direction="row" justifyContent="flex-end" alignItems="flex-start" padding={4}>
-                <Grid item container direction="row" xs={12}>
-                    <MetHeader1 sx={{ mb: 2 }}>Preview Engagement{!isDraft && ' - Published'}</MetHeader1>
+                <ScheduleModal
+                    reschedule={savedEngagement.scheduled_date ? true : false}
+                    open={isOpen}
+                    updateModal={setIsOpen}
+                />
+                <Grid item container direction="row" xs={12} sx={{ pt: 2, mb: 2 }}>
+                    <MetHeader1 sx={{ mb: 2 }}>
+                        {savedEngagement.start_date
```

Comment:
I think we need to check for the submission status/engagement status intead of the the start date